doc/helm/user-guide.md

```diff
@@ -61,7 +61,11 @@ If a custom repository URL is specified by `--helm-chart-repo`, the only support
 - `<chartName>`: Fetch the helm chart named `chartName` in the helm chart repository
                  specified by the `--helm-chart-repo` URL.
 
-If `--helm-chart-version` is not set, the SDK will fetch the latest available version of the helm chart. Otherwise, it will fetch the specified version. `--helm-chart-version` is not used when `--helm-chart` itself refers to a specific version, for example when it is a local path or a URL.
+If `--helm-chart-version` is not set, the SDK will fetch the latest available version of the helm chart. Otherwise, it will fetch the specified version. The option `--helm-chart-version` is not used when `--helm-chart` itself refers to a specific version, for example when it is a local path or a URL. 
+
+For more details and examples see the [Helm CLI reference doc][helm-reference-cli-doc]
+
+```shell
```

Comment:
```suggestion
```

If you check the doc you will see that the shell here will break the format since it is used for we are able to add code.  

<img width="992" alt="Screenshot 2020-01-16 at 09 44 22" src="https://user-images.githubusercontent.com/7708031/72513255-cc048000-3844-11ea-8f65-bb6e97eb3901.png">

Also, could you please add a point to finish the phase added?